package.json

```diff
@@ -98,7 +97,7 @@
     "@commitlint/cli": "^16.2.4",
     "@commitlint/config-conventional": "^16.2.4",
     "@open-wc/webpack-import-meta-loader": "^0.4.7",
-    "@polkadot/types": "9.11.1",
+    "@polkadot/types": "9.14.2",
```

Comment:
@bvotteler do you see any issue with having a newer version of the types? We've always kept these in sync before, and I'm not sure if there will be any knock-on effects. Everything seems to be working, but polkadot versions have caught us out before!

Reply:
I could see that potentially leading to some unexpected errors at runtime if the polkadot api isn't the same version - TypeScript static checks pass, but the actual inputs/outputs could be different.
I can't tell for sure, though. I'd certainly lean towards keeping those versions in lockstep.